src/client/Microsoft.Identity.Client/Platforms/iOS/IosWebUIFactory.cs

```diff
@@ -1,40 +1,41 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-using Microsoft.Identity.Client.ApiConfig.Parameters;
-using Microsoft.Identity.Client.Internal;
-using Microsoft.Identity.Client.Platforms.iOS.EmbeddedWebview;
-using Microsoft.Identity.Client.Platforms.iOS.SystemWebview;
-using Microsoft.Identity.Client.PlatformsCommon.Interfaces;
-using Microsoft.Identity.Client.UI;
-
-namespace Microsoft.Identity.Client.Platforms.iOS
-{
-    internal class IosWebUIFactory : IWebUIFactory
-    {
-        public bool IsSystemWebViewAvailable => true;
-        public bool IsUserInteractive => true;
-        public bool IsEmbeddedWebViewAvailable => true;
-
-        public IWebUI CreateAuthenticationDialog(
-            CoreUIParent coreUIParent, 
-            WebViewPreference useEmbeddedWebView, 
-            RequestContext requestContext)
-        {
-            if (useEmbeddedWebView == WebViewPreference.Embedded)
-            {
-                return new EmbeddedWebUI()
-                {
-                    RequestContext = requestContext,
-                    CoreUIParent = coreUIParent
-                };
-            }
-            
-            return new SystemWebUI()
-            {
-                RequestContext = requestContext
-            };
-        }
-
-    }
-}
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.

```

Comment:
There's smth wrong with your spacing settings. The diffs are all screwed up :)

Reply:
We use the VS defaults around tabs and there is an .editorconfig file that sets some styles for you... 
![image](https://user-images.githubusercontent.com/12273384/117144906-68d88d00-adaa-11eb-9f4a-ff325dbe8e71.png)

Reply:
I have those settings. Not sure what went wrong. Hope it goes fine in the second push.